relay-clients/client-kusama/src/lib.rs

```diff
@@ -126,5 +126,5 @@ impl ChainWithTransactions for Kusama {
 
 impl ChainWithRuntimeVersion for Kusama {
 	const RUNTIME_VERSION: Option<SimpleRuntimeVersion> =
-		Some(SimpleRuntimeVersion { spec_version: 1_002_004, transaction_version: 25 });
+		Some(SimpleRuntimeVersion { spec_version: 1_004_000, transaction_version: 26 });
```

Comment:
@rosarp do not change this, unless you regenerate codegen
```suggestion
		Some(SimpleRuntimeVersion { spec_version: 1_002_004, transaction_version: 26 });
```

I am suggesting just to do `relay-clients/client-bridge-hub-polkadot` now and after release, you can regenerate all runtimes, we submit extrinsics only to BridgeHubs